standalone/rpc/src/lib.rs

```diff
@@ -39,7 +39,7 @@ use sc_client_api::AuxStore;
 use sc_consensus_babe::{BabeConfiguration, Epoch};
 use sc_consensus_epochs::SharedEpochChanges;
 use sc_finality_grandpa::{
-	FinalityProofProvider, GrandpaJustificationStream, SharedAuthoritySet, SharedVoterState,
```

Comment:
This file comes from the upstream.

Reply:
Let me revert it

Reply:
Reverted the fmt. Let's still keep fixing the warning since clippy doesn't support to excluding a crate yet.
https://github.com/rust-lang/rust-clippy/issues/9555